app/models/pusher.rb

```diff
@@ -87,9 +89,12 @@ def find
     # Update the name to reflect a valid case change
     @rubygem.name = name
 
+    sha256 = Digest::SHA2.base64digest(body.string)
```

Comment:
Sha265 hashes are _really_ long as hex. I completely agree with this, and strongly prefer to store them as base64. I plan to store them as base64 in the new index as well.